examples/image_classifier/streaming_video/request_client_batching.py

```diff
@@ -0,0 +1,158 @@
+import argparse
```

Comment:
same comment on refactoring: this file is very similar to the one below it

Reply:
Done